library/spdm_responder_lib/libspdm_rsp_key_exchange.c

```diff
@@ -238,11 +254,8 @@ libspdm_return_t libspdm_get_response_key_exchange(void *context,
         }
     }
 
```

Comment:
Since slot_id maybe 0xFF, do we need similar check above?
E.g. check to ensure spdm_context->local_context.local_public_key_provision

Reply:
No need to.
The slot_id is clearly checked just at the line L224-L239 above.

Reply:
See comment in https://github.com/DMTF/libspdm/pull/1556#discussion_r1062308490

Reply:
Point is taken, I will fix it by next patch set.